package.json

```diff
@@ -10,7 +10,16 @@
     "test": "react-scripts test --watchAll=false",
     "test:coverage": "react-scripts test --coverage --watchAll=false",
     "test:update": "react-scripts test --updateSnapshot --watchAll=false",
-    "coveralls": "react-scripts test --coverage --watchAll=false && cat ./coverage/lcov.info | coveralls"
+    "coveralls": "react-scripts test --coverage --watchAll=false && cat ./coverage/lcov.info | coveralls",
+    "storybook": "start-storybook",
+    "build-storybook": "build-storybook -c .storybook -o .out"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "**/*.{js,jsx}",
+      "!**/node_modules/**",
+      "!**/*.stories.{js,jsx}"
+    ]
```

Comment:
- You can try to add jest setting into a new file .jest.json and import the setting from the setting file
- You can add more exclude files, such as all index.js in the component and pages folders, config file

Reply:
Done. I excluded these files and folders